diff --git a/modules/defaultServices.jsm b/modules/defaultServices.jsm
index 63c68bf..6a15446 100644
--- a/modules/defaultServices.jsm
+++ b/modules/defaultServices.jsm
@@ -47,6 +47,13 @@ function frecencyForUrl(host)
 // be maintained in the case the user does not have any logins or frecency.
 // the mediator will enable via login and frecency, and sort by frecency.
 var builtins = [
+  {
+    name: "Yammer",
+    iconURL: "https://www.yammer.com/favicon.ico",
+    workerURL: "resource://socialdev/providers/yammer/worker.js",
+    sidebarURL: "resource://socialdev/providers/yammer/yammer.html",
+    URLPrefix: "https://www.yammer.com",
+  }
 ];
 
 function installBuiltins() {
